[BUGFIX]: scipy.misc.imresize is removed from latest version of SciPy

yngvem opened the pull request:
The documentation did not build with latest version of SciPy since `scipy.misc.imresize` is removed as of SciPy v.1.3.0. However, `scipy.ndimage.zoom`, which can accomplish the same with slightly different syntax, is. 

This pull request only includes a fix to this problem, and contains one new line and one changed line.

JeanKossaifi commented:
Great, thanks @yngvem !

Why the addition of the `squeeze` in `to_image` -- Is there a case where it is needed? `zoom` doesn't add a dimension, right?

yngvem commented:
No, I thought I removed that part! I removed it now.

I added that line when I looked for a bug that arose by the `zoom` function needing a tuple as input instead of a float. Otherwise, the `zoom` function shrinks the channel mode as well as the spatial modes of the image tensor.

JeanKossaifi commented:
Great, merging! Thanks @yngvem !
